conversations/v3/whatsapp.yaml

```diff
@@ -632,8 +632,9 @@ paths:
   /channels/whatsapp/phone-numbers/{phone-number}/settings/logo:
     parameters:
       - $ref: '#/components/parameters/PhoneNumber'
-      - $ref: '#/components/parameters/WhatsAppProfileLogoFormat'
     get:
+      parameters:
+        - $ref: '#/components/parameters/WhatsAppProfileLogoFormat'
       operationId: getCurrentWALogo
```

Comment:
The idea here is that the `PUT` operation does not use the `format` parameter. Is that right?

Reply:
Yes, that's correct. As for updates on the logo you just upload the binary file